nys-nydanske-studier.csl

```diff
@@ -0,0 +1,431 @@
+          <text variable="container-title" font-style="italic"/>
+          <text macro="collection-title"/>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="publisher">
+    <group delimiter=": ">
+      <text variable="publisher-place"/>
+      <text variable="publisher"/>
+    </group>
+  </macro>
+  <macro name="date">
+    <date variable="issued">
+      <date-part name="year"/>
+    </date>
+  </macro>
+  <macro name="collection-title">
+    <choose>
+      <if variable="collection-title">
+        <group prefix="(" suffix=")">
+          <text variable="collection-title" text-case="title"/>
+          <text variable="collection-number" prefix=" "/>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="event">
+    <choose>
+      <if variable="container-title" match="none">
+        <choose>
+          <if variable="event">
+            <choose>
+              <if variable="genre" match="none">
+                <text term="presented at" text-case="capitalize-first" suffix=" "/>
+                <text variable="event"/>
+              </if>
+              <else>
+                <group delimiter=" ">
+                  <text variable="genre" text-case="capitalize-first"/>
+                  <text term="presented at"/>
+                  <text variable="event"/>
+                </group>
+              </else>
+            </choose>
+          </if>
+          <else-if type="speech">
+            <text variable="genre" text-case="capitalize-first"/>
+          </else-if>
+        </choose>
+      </if>
+    </choose>
+  </macro>
+  <macro name="description">
+    <group delimiter=". ">
+      <text macro="interviewer"/>
+      <text variable="medium" text-case="capitalize-first"/>
+    </group>
+    <choose>
+      <if variable="title" match="none"/>
+      <else-if type="thesis speech" match="any"/>
+      <else>
+        <text variable="genre" text-case="capitalize-first"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="issue">
+    <choose>
+      <if type="speech">
+        <group delimiter=", ">
+          <text macro="event"/>
+          <text variable="event-place"/>
+        </group>
+      </if>
+      <else-if type="thesis">
+        <group delimiter=" ">
+          <text macro="publisher"/>
+          <text variable="genre"/>
+        </group>
+      </else-if>
+      <else>
+        <group delimiter=", ">
+          <text macro="publisher"/>
+          <choose>
+            <if type="manuscript">
+              <text value="ms"/>
+            </if>
+          </choose>
+        </group>
+      </else>
+    </choose>
+  </macro>
+  <macro name="chapter-prefix">
+    <choose>
+      <if type="chapter paper-conference" match="any">
+        <text term="in" text-case="capitalize-first"/>
+      </if>
+    </choose>
+  </macro>
+  <citation et-al-min="4" et-al-use-first="1" disambiguate-add-year-suffix="true" disambiguate-add-names="true" disambiguate-add-givenname="true">
```

Comment:
This should be `et-al-min="3"` as per 5.9 p.6 of the guide, no?
More generally, I'd recommend removing all et al from the various places it's in `names` and at the top of the style and _only_ using it on `citation` and `bibliography` to improve readability and reliability of the style.

Reply:
Man, you are observant! I am impressed! 

But no, for some reason I can't explain, the style stipulates et-al-min=3 in in-line references (that's §5.9 as you referenced), but et-al-min=4 in the reference list. The latter is explained in §6.1 (bullet 3.3): "Ved fire eller flere forfattere/redaktører bruges ’m.fl.’ eller ’et al.’ efter første forfatter- eller redaktørnavn (fx Everaert, M.B.H. et al. 2015)" [in case of four or more authors/editors, use 'm.fl.' or 'et.al.' after the first author or editor name (e.g. Everaert, M.B.H. et al. 2015.)].

Can you help me with the most efficient and readable way to get this into the style? I would REALLY appreciate any help you can give. I know one or two things about editing a journal, but writing code is not my forte... I'll of course give it a shot if this is beyond the services you provide. But chances are, I'll only be making more inconsistencies...

In gratitude,
Jacob